querydsl-codegen/src/test/java/com/querydsl/codegen/GenericExporterTest.java

```diff
@@ -117,5 +117,11 @@ public void Export_Domain_Package() {
         exporter.export(Cat.class.getPackage());
     }
 
+    @Test
+    public void Export_SerializerConfig() {
+        exporter.setTargetFolder(new File("target/gen7"));
+        exporter.setSerializerConfig(new SimpleSerializerConfig(true, true, true, true, ""));
+        exporter.export(Cat.class.getPackage());
```

Comment:
Do you want to introduce helper methods for that?
Getting contents of a file in Java requires a lot of boilerplate code, if you want to introduce content assertions it's good to get it right the first time.
It's then also useful for other tests to benefit from the scaffolding.